test/e2e/nodenetworkstate_test.go

```diff
@@ -30,8 +42,41 @@ var _ = Describe("NodeNetworkState", func() {
           stp-path-cost: 100
           stp-priority: 32
 `)
-			)
+		br1WithBond1Up = nmstatev1alpha1.State(`interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+  - name: bond1
+    type: bond
+    state: up
+    link-aggregation:
+      mode: active-backup
+      slaves:
+        - eth1
+      options:
+        miimon: '120'
+  - name: br1
+    type: linux-bridge
+    state: up
+    bridge:
+      options:
+        stp:
+          enabled: false
+      port:
+        - name: bond1
+          stp-hairpin-mode: false
+          stp-path-cost: 100
+          stp-priority: 32
+`)
 
+		br1Absent = nmstatev1alpha1.State(`interfaces:
```

Comment:
what about absent bonds? do we cleanup connections on host (is there leftover of deactivated connection)?

Reply:
I leave it on purpose, as it does exactly same thing as bridges just link del.

Reply:
i don't understand.

why do we have it for bridge and not for bond?

do we have leftover connections on node? if so, it affects the test. in real situation, we would have no pre-existing connection. but in this test, we reuse the same over and over again (wild guess)